modules/tinymce/src/themes/silver/main/ts/alien/NotificationManagerImpl.ts

```diff
@@ -79,6 +79,8 @@ export default (
       });
     };
 
+    const shouldApplyDocking = () => !Options.isStickyToolbar(editor) || !sharedBackstage.header.isPositionedAtTop();
```

Comment:
Could this use `comp.hasConfigured(Docking)` instead? We always want to refresh if it's docked right?